### ISSUE TITLE
--protect-args unknown option on OS X

### ISSUE THREAD

**Issue by grahamgilbert** [cyaninc/git-fat#52]
When running `git fat push` on OS X 10.10.3, the following error is produced:

```
rsync: --protect-args: unknown option
rsync error: syntax or usage error (code 1) at /SourceCache/rsync/rsync-45/rsync/main.c(1333) [client=2.6.9]
```

Removing `--protect-args` from line 379 resolves this, but I'm unfamiliar with this option so I'm not sure what the effect of this is.

**Comment by justinclift** [comment]
Which version of git fat are you using? :smile:  Asking just in case you're using an older version, as I kind of remember something about the guys working on this problem for the release that was done recently.

**Comment by grahamgilbert** [comment]
The current master: https://github.com/cyaninc/git-fat/blob/develop/git_fat/git_fat.py#L379

**Comment by abraithwaite** [comment]
Yep, this is related to a change merged a while back (not this latest release though)

https://github.com/cyaninc/git-fat/pull/35#issuecomment-74101342

I'll keep this issue open, but right now the easiest and probably best thing to do if you're able to is install a later version of rsync from [`homebrew`](http://brew.sh/) or [`macports`](https://www.macports.org/).

**Comment by natewalck** [comment]
Would it be possible to add --protected-args based on rsync version? OS X 10.11 ships with rsync 2.6.9, which is ancient.

There was another commit on #35 that said it fixed this, but it still seems broken on master.

**Comment by natewalck** [comment]
I just pulled down master to test and it looked like there is another error now being thrown (on 10.11.1 at least):

```
rsync: on remote machine: -sRe.Lsfx: unknown option
```
